### PR TITLE
Add follow option

### PR DESCRIPTION
**Pull request by JoshHeng**
## What's Changing

Add follow option to add the --follow parameter when executing `git log`

## Change Type

Indicate the type of change your pull request is:

- [ ] `patch`
- [X] `minor`
- [ ] `major`

### CODE REVIEW

**Comment by JoshHeng** [comment]
Hey! I'm working on a project which uses `gitlog` and have added an option to add the `--follow` parameter to be passed into the command line. This means that it will correctly track previous contributors to the a file, even if it has been renamed.

If this could be reviewed/merged and then a new package built on npm that would be great. Thank you!

File: README.md [hunk 1]
```diff
@@ -105,6 +105,12 @@ Pass the `-m` option to includes files in a merge commit.
 
 This option is disabled by default.
 
+### follow
+
+Pass the -follow option to follow files across renames.
```
Comment (domharrington) [inline README.md:110]:
```suggestion
Pass the --follow option to follow files across renames.
```

**Review by domharrington — APPROVED** [review]
Hey! Thank you for your contribution! I've just approved the tests to run, so if they pass then i'll get this merged and released for you. Thanks.

**Comment by domharrington** [comment]
Published to npm as v5.1.0: https://www.npmjs.com/package/gitlog/v/5.1.0

Thanks again!

**Comment by JoshHeng** [comment]
Thank you!

**Pull request merged**